tests/integration/defs/perf/nvdf.py

```
@@ -0,0 +1,578 @@
+    "d_p99_ttft",
+    "d_mean_itl",
+    "d_median_itl",
+    "d_p99_itl",
+    "d_mean_e2el",
+    "d_median_e2el",
+    "d_p99_e2el",
+]
+
+
+def get_nvdf_config() -> dict:
+    gpu_type, gpu_count, build_id, build_url, job_name, job_id, job_url,
```

Ruff (0.14.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)